Source/PLCrashAsyncLinkedList.hpp

```diff
@@ -80,12 +84,14 @@ class async_list {
         }
 
     private:
+        std::atomic<node> _a;
```

Comment:
What is it? Why do you add it instead of replace existing field to just resolve a warning? Naming, comments